Sources/EpoxyCore/Model/Providers/ErasedContentProviding.swift

```diff
@@ -14,7 +14,7 @@ public protocol ErasedContentProviding {
   /// A closure that can be called to determine whether the given `model`'s `erasedContent` is equal
   /// to this model's `erasedContent`, else `nil` if there is no content or the content is always
   /// equal.
-  var isErasedContentEqual: ((Self) -> Bool)? { get }
+  var isErasedContentEqual: ((ErasedContentProviding) -> Bool)? { get }
```

Comment:
This is a very subtle change that could affect implementations, since now we don't guarantee at compile-time that the types we are comparing match. It would be nice if we did a search for how we are using this property internally to validate that we think this change is safe.